### ISSUE TITLE
Mulighed for at redigere tegnet geometri

### ISSUE THREAD

**Issue by MichaelDavidKnudsen** [Septima/OneDoor#91]
Mulighed for at redigere tegnet geometri

**Comment by rastermanden** [comment]
Der har også været ønsker om en funktion til at vælge geometrier i kortet. Det kan f.eks bruges til at lave geometri til en høringsliste

**Comment by MichaelDavidKnudsen** [comment]
Ja, det vil også være interessant, at kunne vælge eksisterende geometrier i kortet; eksempelvis en lokalplan eller matrikler.

Og måske også en mulighed for at lægge en negativ buffer på den valgte geometri, hvilket i nogle tilfælde er nødvendigt for at få en retvisende geometri til en høring.

**Comment by rastermanden** [comment]
Hvis vi kan finde en god featurelist, kan vi lave en "crowdfunding" her på Github

**Comment by DocDemar** [comment]
Vi kommer med et første bud på en feature liste og estimat i løbet af august

____ Feature list  _____

- Snap
- Buffer
- Udpeg objekter i kort
